File: Magas Szintű Programozási Nyelvek 2/Könyv/source/2.Liskov/Parentchild/szülőgyerek.java
class Killer
	{
	public void death ()
		{
		}
	};

class SerialKiller extends Killer
	{
	public void serialkillerdeath ()
		{
		}
	};

class Program
	{
	public static void main(String[] args)
		{
		Killer killer= new Killer ();
		SerialKiller serialkiller= new SerialKiller ();

		serialkiller.death ();
		serialkiller.serialkillerdeath ();

		killer.death ();

		//Az alsó sort kikommentelve, lefut a programunk, viszont ha nem kommenteljük ki, akkor ugyan olyan hibakódot kapunk
		// mint a C++ -os változatban.

		//killer.serialkiller();

		System.out.print("\"ÉÉéés hát itt is megjött a gyilkos. :O Javában is itt van :O\"");
		}
	};

File: Magas Szintű Programozási Nyelvek 2/Könyv/source/2.Liskov/Parentchild/szülőgyerek.cpp
#include <iostream>
using namespace std;

class Killer
	{
	public:
	void death(){};
	};

class SerialKiller : public Killer 
	{ 
	public:
	void serialkillerdeath(){};
	};

int main(void)
	{
	Killer* killer;
	SerialKiller* serialkiller; 

	(*serialkiller).death(); 
	(*serialkiller).serialkillerdeath();

	(*killer).death();

	std::cout << "Ééééés lefutott, megjött a gyilkos. :O" << endl;
	};

